main.py

```diff
@@ -107,23 +107,27 @@ def main(username, scope, clientID, clientSecret, redirectURI, path):
         spotify_client_id = input("Great, now what's the ClientID you're using? ")
         spotify_client_secret = input("Beautiful, now how about the Client Secret? ")
 
-        save = input("Awesome, now would you like to save these settings for future sessions? (Y/n) ")
-        
-        if save == "Y":
+        save = input("Awesome, now would you like to save these settings for future sessions? (y/N) ")
+
+        if save.lower() == "y":
             save_obj = {
                 "spotify_username": spotify_username,
                 "spotify_client_id": spotify_client_id,
                 "spotify_client_secret": spotify_client_secret
             }
 
-            with open("./credentials.json", "w") as creds:
-                creds.write(json.dumps(save_obj))
-                creds.close()
+            try:
+                with open("./credentials.json", "w") as creds:
+                    creds.write(json.dumps(save_obj))
+                    creds.close()
 
-                print("Saved.")
+                    print("Saved.")
+            except Exception:
```

Comment:
Please don't catch broad exceptions if the exception type that *might* occur is known. In this case, it will probably be an OSError. If you do want to catch a broad exception, make sure you show the user what the error was:

```python
try:
  # Whatever you do here
except Exception as e:
  print(f'Something went wrong: {e}')
```